### PR TITLE
Reduce logging to help loki

### PR DESCRIPTION
**Pull request by n-hutton**
To much logging has caused loki to run out of space - further reduce the logging.

### CODE REVIEW

File: libs/ledger/src/storage_unit/transaction_store_sync_protocol.cpp [hunk 1]
```diff
@@ -211,6 +205,12 @@ TSSP::TxArray TransactionStoreSyncProtocol::PullSpecificObjects(DigestSet const
     }
   }
 
+  if (!ret.empty())
```
Comment (n-hutton) [inline libs/ledger/src/storage_unit/transaction_store_sync_protocol.cpp:208]: This should be here (printing out if there is a specific pull needed, not normal pull)

File: libs/ledger/src/storage_unit/transaction_store_sync_service.cpp [hunk 2]
```diff
@@ -463,6 +463,7 @@ TransactionStoreSyncService::State TransactionStoreSyncService::OnResolvingObjec
   if (counts.failed != 0u)
   {
     FETCH_LOG_WARN(LOGGING_NAME, "Lane ", cfg_.lane_id, ": ", "Failed promises: ", counts.failed);
+    pending_objects_.DiscardFailures();
```
Comment (n-hutton) [inline libs/ledger/src/storage_unit/transaction_store_sync_service.cpp:466]: I believe this should stop it printing out how many failures there are forever.

Comment (ejfitzgerald) [reply libs/ledger/src/storage_unit/transaction_store_sync_service.cpp:466]: 👍

File: libs/ledger/src/miner/basic_miner.cpp [hunk 3]
```diff
@@ -219,8 +219,8 @@ void BasicMiner::GenerateBlock(Block &block, std::size_t num_lanes, std::size_t
   std::size_t const remaining_transactions = mining_pool_.size();
   std::size_t const packed_transactions    = pool_size_before - remaining_transactions;
 
-  FETCH_LOG_INFO(LOGGING_NAME, "Finished block packing (packed: ", packed_transactions,
-                 " remaining: ", remaining_transactions, ")");
+  FETCH_LOG_DEBUG(LOGGING_NAME, "Finished block packing (packed: ", packed_transactions,
```
Comment (n-hutton) [inline libs/ledger/src/miner/basic_miner.cpp]:
```suggestion
  FETCH_LOG_INFO(LOGGING_NAME, "Finished block packing (packed: ", packed_transactions,
```

**Review by ejfitzgerald — DISMISSED** [review]

File: libs/ledger/src/miner/basic_miner.cpp [hunk 4]
```diff
@@ -220,7 +220,7 @@ void BasicMiner::GenerateBlock(Block &block, std::size_t num_lanes, std::size_t
   std::size_t const packed_transactions    = pool_size_before - remaining_transactions;
 
   FETCH_LOG_INFO(LOGGING_NAME, "Finished block packing (packed: ", packed_transactions,
-                 " remaining: ", remaining_transactions, ")");
+                  " remaining: ", remaining_transactions, ")");
```
Comment (n-hutton) [inline libs/ledger/src/miner/basic_miner.cpp]:
```suggestion
                 " remaining: ", remaining_transactions, ")");
```

**Review by ejfitzgerald — APPROVED** [review]

**Pull request merged**